README.md

```diff
@@ -1,128 +1,164 @@
-
-### Target Audience
-
-The app targets shoppers across Australia who are looking for unique and quality Japanese food products, including:
-
-- Japanese people residing in Australia
-- Japanese food products lovers
-
-## Application Architecture Diagram
-![Application-Architecture-Diagram](./docs/images/application-architecture-diagram/application_architecture_diagram.png)
-
-## User Stories 
-
-Working in progress...
-
-## Functionality/Features
-
-The MVP includes two parts: the static website and the admin dashboard. The overall function of the website is to operate as an online store for selling Japanese food products.
-
-### Features
-
-- User authentication and authorisation (Register and Login/Logout)
-- Sign-up confirmation via email
-- Saved login details
-- Site navigation
-- Product search by name and by categories
-- Product details view
-- Showcase of new arrivals and featured products
-- Shopping cart checkout
-- Calculation for different delivery options
-- Billing services
-- Order confirmation via email
-- Contact form
-- User account view
-- Admin dashboard:
-  - User management
-  - Product management
-  - Order management (nice to have)
-
-### Functionality
-
-#### Users
-
-- Website visitors without authentication can view products.
-- Website visitors can add products to cart and will be prompted to register or log in if they wish to proceed to payment.
-- Users with authentication can perform CRUD operations on personal account information.
-- Users with authentication can perform CRUD operations on product orders.
-- Users with authentication can perform CRUD operations on billing and delivery information.
-
-#### Admins
-
-- Admins can perform CRUD operations on users' information.
-- Admins can perform CRUD operations on products.
-- Admins can perform CRUD operations on product categories.
-- A nice-to-have functionality: Admins can have the following operations on product orders, including search orders, view orders, edit orders and track orders.
-
-## Sitemap
-
-![Sitemap](./docs/images/sitemap/sitemap.png)
-
-## Tech Stack
-
-The core tech stack is MERN stack.
-
-- Application:
-  - Back-end API: NodeJS, ExpressJS, Mongoose
-  - Front-end: HTML, CSS, JavaScript, ReactJS, Axios, Tailwind, NextUI
-- Database:
-  - MongoDB, MongoDB Atlas
-- Testing:
-  - Jest, Supertest
-- Deployment:
-  - Back-end API: Heroku
-  - Front-end: Netlify
-- DevOps:
-  - Git
-  - GitHub
-  - VS Code
-- Project Management:
-  - Trello
-  - Discord
-  - Skype
-- Design Tools:
-  - Draw.io
-  - Miro
-
-## Tasks planning and tracking
-
-View the description of the way tasks are allocated and tracked in the project [here](./task_tracking.md).
-
-Link to project management tool [Trello](https://trello.com/b/TE5Q9ZYj/t3a2-%F0%9F%8C%B8sakura-pantry).
+## T3A2 Full Stack App

```

Comment:
![Screenshot 2023-11-27 at 9 32 25 pm](https://github.com/irene2mana-T3A2-2023/SakuraPantryDocs/assets/98362650/15c51df8-2c22-4653-a9de-60d4178473a6)

The first line in a file should be at the top level. Could you please make this change as well?